### ISSUE TITLE
Problem with item flag countable

### ISSUE THREAD

**Issue by Genschere** [PokeAPI/pokeapi#1047]
Might be a minor thing but I wondered what the item flag (item_flags.csv) _countable_ actually means. It seems hard to misunderstand but if it really means items that can have a count of how many there are, the data is quite bad on this.
As example: all pokeballs are countable but berries are not. Actually most items that have a count in the games are not flagged as countable.
I wonder if there is a special reason for it or if nobody bothered setting it because almost everything can have a count anyway. The only items that are non-countable are actually key and story items with special purpose like the bike.

Would it not make more sense to change this to a _non-countable_ flag and only mark these "few" items with it?
I know TMs are a bit more complicated on this since it depends on the generation if the item is unique or can have a count but since TMs are a bit special anyway this shouldn't be a reason to not correct this.

**Comment by GreatNovaDragon** [comment]
IMO, the flag countable, or an counterflag uncountable, isnt needed, cause the non-countables should be base items, any other item is countable, with the exception of TMs between gen5-7 and HMs sofar they existed, but the data displays the most current data. 
one could make their counterability based upon categoric exclusion in code.

**Comment by Genschere** [comment]
True, but why does it even exist then?
It could simply be removed.

**Comment by GreatNovaDragon** [comment]
I muse, cause its an carryover from veekun/pokedex now outdated models

**Comment by Naramsim** [comment]
Yep, most surely is an outdated model/piece of data. The prose says _Has a count in the bag_, so berries should be countable. And maybe every item, nowadays, but I don't know anything about gens past 3rd. 

We could update the data and reflect the current real values, or flip the data.
